darwin/importer/formats/superannotate.py

```diff
@@ -159,7 +159,7 @@ def _to_keypoint_annotation(point: Dict[str, Any], classes: List[Dict[str, Any]]
     class_id: int = cast(int, point.get("classId"))
 
     name = _find_class_name(class_id, classes)
-    return make_keypoint(name, x, y)
+    return make_keypoint(name + "-point", x, y)
```

Comment:
Rather than concatenation, I'd go for interpolation in these cases, as we do in other places in the codebase
```suggestion
    return make_keypoint(f"{name}-point", x, y)
```

Reply:
done !